tests/layer_tests/onnx_tests/test_fusedgemm.py

```diff
@@ -0,0 +1,153 @@
+# Copyright (C) 2018-2022 Intel Corporation
+# SPDX-License-Identifier: Apache-2.0
+
+import os
+
+import numpy as np
+import pytest
+import torch
+from common.layer_test_class import CommonLayerTest
+from common.onnx_layer_test_class import OnnxRuntimeLayerTest
+
+
+class TestFusedGemm(OnnxRuntimeLayerTest):
```

Comment:
Primary tests for operations in onnx importer are here: https://github.com/openvinotoolkit/openvino/blob/master/src/frontends/tests/onnx/onnx_import.in.cpp

Could you add some tests like that?
(reference PR with such tests as example: https://github.com/openvinotoolkit/openvino/pull/11766/files)

Reply:
ok, i will try to add fusedgemm test into the right place.

Reply:
I have added the test code into https://github.com/openvinotoolkit/openvino/blob/master/src/frontends/tests/onnx/onnx_import_com_microsoft.in.cpp